gilbreth_moveit_config/config/gilbreth.srdf

```diff
@@ -10,10 +10,10 @@
     <!--CHAINS: When a chain is specified, all the links along the chain (including endpoints) are included in the group. Additionally, all the joints that are parents to included links are also included. This means that joints along the chain and the parent joint of the base link are included in the group-->
     <!--SUBGROUPS: Groups can also be formed by referencing to already defined group names-->
     <group name="robot_rail">
-        <chain base_link="linear_actuator_base_link" tip_link="robot_tool0" />
+        <chain base_link="linear_actuator_base_link" tip_link="vacuum_gripper_tool0" />
```

Comment:
Suggestion: don't use `tool0` for actual tool frames. `tool0` is supposed to represent (from [Coordinate Frames for Industrial Robots](https://github.com/gavanderhoorn/rep/blob/coord_frames_ind_robots/rep-xxxx.rst), a REP I'm writing):

>  an all-zeros Tool Centre Point (TCP) as defined by the robot controller. For most controllers, an all-zeros TCP is equal to an unconfigured (or default) TCP, which typically lies on the robot's physical mounting flange.

So unless the vacuum gripper has an "unconfigured (or default) TCP", the `tool0` suffix should not be used.

Semantically it would be much better if you give the frame a name that conveys what it is for (ie: the fact that it is a tool frame defined for the vacuum gripper (or suction cup)).